docs-client/src/lib/transports/annotated-http.ts

```diff
@@ -86,6 +86,7 @@ export default class AnnotatedHttpTransport extends Transport {
     bodyJson?: string,
     endpointPath?: string,
     queries?: string,
+    pathPrefix?: string,
```

Comment:
Can't this be `pathPrefix` because we specify an empty string when it's null?

Reply:
Yep.